### PR TITLE
3-GET/api/topics

### PR DESCRIPTION
**Pull request by Adedunsola**
#3. GET /api/topics

### CODE REVIEW

File: __tests__/app.test.js
Comment (leekli) [inline __tests__/app.test.js:48]: Good test, your checking the structure and shape of the data returned to ensure it fits the structure you're expecting

File: __tests__/app.test.js
Comment (leekli) [inline __tests__/app.test.js:58]: Brill, good to see you testing for some errors

File: app.js
Comment (leekli) [inline app.js]: You probably don't need a console log here as you're dealing with the response by sending a response to the user, I'd recommend removing this console log.

File: controllers/errors.controller.js
Comment (leekli) [inline controllers/errors.controller.js]: I shouldn't see commented out code as part of a commit which is meant to be production-ready, please remove this

File: controllers/topics.controller.js
Comment (leekli) [inline controllers/topics.controller.js:10]: Good naming conventions for your controller and model - well done!

File: controllers/topics.controller.js
Comment (leekli) [inline controllers/topics.controller.js:12]: response being sent back on a keyword of 'topics' is spot on, great work

**Review by leekli — CHANGES_REQUESTED** [review]
Hi Ade,

Great work on this ticket. I have left comments around the commit for you to read.

It's not quite ready to be merged yet, there are things like a console log, commented out code which need removing to make this commit 'production ready'.

Please review these comments, make the necessary changes and then submit a new pull request for review. Please do not merge this branch until we have approved it after the next pull request with the requested changes made.

**Review by RichardNeat — COMMENTED** [review]
Great merge away :)

**Pull request merged**
